core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -40,19 +40,9 @@
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 
-import java.io.ByteArrayInputStream;
-import java.io.EOFException;
-import java.io.FileNotFoundException;
-import java.io.FilterInputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
This and other wildcard imports will either eventually be rejected, or cause merge conflicts from #15395.